index.js

```diff
@@ -32,7 +32,8 @@ module.exports = function(opts){
     if (user && user.name == opts.name && user.pass == opts.pass) {
       return next();
     } else {
-      ctx.throw(401);
+      ctx.set('WWW-Authenticate', 'Basic realm="Secure Area"');
+      ctx.status = 401;
```

Comment:
1. remove `ctx.throw` is broken.

2. we can define an option called `realm`, and add a default value, such as: `'Authorization Required'`, then `set('WWW-Authenticate', 'Basic realm="' + realm + '"')`

3. we need add tests for this :)